src/wallet/wallet.cpp

```diff
@@ -3728,7 +3728,7 @@ void CWallet::GetFilteredNotes(std::vector<CNotePlaintextEntry> & outEntries, st
  * These notes are decrypted and added to the output parameter vector, outEntries.
  */
 void CWallet::GetFilteredNotes(
-    std::vector<CNotePlaintextEntry>& outEntries,
+    std::vector<CSproutNotePlaintextEntry>& outEntries,
```

Comment:
This will either be reverted once we add Sapling notes, or we'll add `outSaplingEntries`.